servicetalk-examples/grpc/debugging/src/main/java/io/servicetalk/examples/grpc/debugging/DebuggingClient.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © 2022 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.examples.grpc.debugging;
+
+import io.servicetalk.grpc.netty.GrpcClients;
+import io.servicetalk.http.netty.HttpProtocolConfigs;
+
+import io.grpc.examples.debugging.Greeter.BlockingGreeterClient;
+import io.grpc.examples.debugging.Greeter.ClientFactory;
+import io.grpc.examples.debugging.HelloReply;
+import io.grpc.examples.debugging.HelloRequest;
+
+import static io.servicetalk.logging.api.LogLevel.TRACE;
+
+public final class DebuggingClient {
```

Comment:
Consider adding some javadoc and comments for the code to clarify different debugging options and what they mean, similar to what you did for an HTTP example

Reply:
May be worth demonstrating `AsyncContext.disable()` here too